src/Components/CardboardModal/CardboardModal.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import {
+    classNamesFunction,
+    useTheme,
+    styled,
+    DefaultButton,
+    Icon,
+    Modal,
+    PrimaryButton,
+    Stack,
+    IStackTokens
+} from '@fluentui/react';
+import { useId } from '@fluentui/react-hooks';
+import {
+    ICardboardModalProps,
+    ICardboardModalStyleProps,
+    ICardboardModalStyles
+} from './CardboardModal.types';
+import { getStyles } from './CardboardModal.styles';
+import { useTranslation } from 'react-i18next';
+
+const stackTokens: IStackTokens = { childrenGap: 8 };
+
+const getClassNames = classNamesFunction<
+    ICardboardModalStyleProps,
+    ICardboardModalStyles
+>();
+
+const CardboardModal: React.FC<ICardboardModalProps> = (props) => {
```

Comment:
Don't you think it is more like Dialog than Modal?

Reply:
hmm no. this is a modal in my book. Dialog is more like the "Are you sure" type windows we see. Modals are the bigger version that show lots of rich content.
![image](https://user-images.githubusercontent.com/57726991/193328948-d81a1b5e-f6dc-473e-98c3-bc406a60fdc1.png)

![image](https://user-images.githubusercontent.com/57726991/193328991-0e2fd5e4-eb4b-4869-b2cb-b4e7ea0c152e.png)